3573/3/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+  post '/comments' do
+    Comment.create(text: params[:comment],
```

Comment:
too easy, how about handling errors?

Reply:
You can't trust users, so they can put some stuff which will lead to not creating of comment, and it means that you'll fail on this line. So you should check that comment was created and proceed or check that it was invalid and say about this to user.